doc/internal/man3/OSSL_TIME.pod

```diff
@@ -0,0 +1,96 @@
+=pod
+
+=head1 NAME
+
+OSSL_TIME, OSSL_TIME_SECOND, OSSL_TIME_INFINITY,
+ossl_time_now, ossl_time_time_to_timeval, ossl_time_compare,
+ossl_time_add, ossl_time_subtract
+- times and durations
+
+=head1 SYNOPSIS
+
+ #include "internal/time.h"
+
+ typedef OSSL_TIME;
```

Comment:
This line doesn't seem to add anything. I'd remove it.

Reply:
Isn't this type the entire point of this documentation?
Without this line, what is an `OSSL_TIME`?

I didn't want to tie us down to a specific type, it might need to be a structure rather than an integer at some point...

Reply:
> Without this line, what is an OSSL_TIME?

We don't do this for most other types, so why are we making an exception for this one?  Most of our types are opaque structures, but we don't specify that in the documentation.

Reply:
It's internal documentation, it's not critical until it's made public :smile:

Reply:
Ok - fair enough.